ci/snap/ci-deploy.sh

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env bash
 
-set -e -o pipefail -u
+set -e -o pipefail -u -x
+
+stty size
```

Comment:
Okay... so the value of `width` is ultimately extracted by `snapcraftctl` from `stty`? 🤔

Reply:
No way to make `snapcraft` **dis**interested in terminal completely? Like idk, piping the output through `| cat`